apps/condo/domains/marketplace/schema/Invoice.js

```diff
@@ -67,6 +68,7 @@ const sendPush = async ({ originalInput, userId, propertyId, unitName, unitType,
         })
 
         const uniqKey = `marketplace_invoice_published_${updatedItem.id}`
+        const uniqKeyWithTicket = `marketplace_invoice_published_${updatedItem.ticket}`
         if (resident) {
```

Comment:
to simplify code reading:

```suggestion
        if (!resident) return
```